### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by peninah98**
# Description
Fragen section

## How Has This Been Tested and How are your changes run?
To review this project, first see the output screenshot below
clone the project to review it in local machine here is the [link](https://github.com/Nezerwa/Ojemba-VLM-Project)
Run `npm run build` to run the project in locally

## Screenshots of output:
image

### CODE REVIEW

**Review by ell-sie — COMMENTED** [review]
The footer looks Great !

File: images/DSC_7863.jpg
Comment (alainmigabo) [inline images/DSC_7863.jpg:1]: Please Remove this image

**Review by alainmigabo — CHANGES_REQUESTED** [review]

**Comment by alainmigabo** [comment]
Please resolve the conflicts then

**Review by alainmigabo — APPROVED** [review]

**Review by ell-sie — APPROVED** [review]
Well done !

**Pull request merged**